tests/lib/snaps/classic/meta/snap.yaml

```diff
@@ -0,0 +1,6 @@
+name: classic
```

Comment:
Probably best to name it classic-confined or similar, to avoid confusion with that other classic snap.